From 4c0c4b768eb21558d687200aaa2dd75fc70163a9 Mon Sep 17 00:00:00 2001
From: "buluess.li" <buluess.li@rock-chips.com>
Date: Tue, 19 Sep 2017 10:39:22 +0800
Subject: [PATCH] ARM: rockchip_linux_defconfig: enable
 PHY_ROCKCHIP_INNO_HDMI_PHY

hdmi driver need enable PHY_ROCKCHIP_INNO_HDMI_PHY

Change-Id: I4172da24bd0c3200eaa42f73707701581269527f
Signed-off-by: Xinhuang Li <buluess.li@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 28e8c8078413..f0567c12bb7e 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -421,6 +421,7 @@ CONFIG_PWM_ROCKCHIP=y
 CONFIG_PHY_ROCKCHIP_USB=y
 CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_PHY_ROCKCHIP_DP=y
+CONFIG_PHY_ROCKCHIP_INNO_HDMI_PHY=y
 CONFIG_ANDROID=y
 CONFIG_NVMEM=y
 CONFIG_ROCKCHIP_EFUSE=y
-- 
2.35.3

